examples/distil_bert_server_pytorch.py

```diff
@@ -1,3 +1,17 @@
+# Copyright 2022 MOSEC
```

Comment:
Usually I saw the format be like `Copyright 2022 MOSEC Authors. All Rights Reserved.` for the first line?

Reply:
This is auto-generated by the tool `addlicense`.

Check other projects:
* https://github.com/kubernetes/kubernetes/blob/master/cmd/kubectl/kubectl.go
* https://github.com/falconry/falcon/blob/master/falcon/routing/compiled.py

It seems they don't have `All rights reserved. BTW, I don't really know the legal difference.

Reply:
An example: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/allocator.h#L1

Anyway, I think the exact format is not particularly important. However, `MOSEC` is the project name, at least we need to change it to `MOSEC Authors`.

Reply:
Fixed